lib/tasks/detailed_guides.rake

```diff
@@ -10,21 +10,21 @@ namespace :detailed_guide do
         new_slug = "guidance/#{old_slug}"
 
         begin
-          path = Rails.application.url_arbiter_api.reserve_path("/#{new_slug}",
+          path = Rails.application.publishing_api.put_path("/#{new_slug}",
             "publishing_app" => "whitehall")
           detailed_guide.set(:slug, new_slug)
           puts "Renamed #{old_slug} to #{new_slug} (#{i+1}/#{count})"
-        rescue GOVUK::Client::Errors::Conflict => e
+        rescue GdsApi::HTTPClientError => e
           puts "Couldn't rename #{old_slug} to #{new_slug} (#{i+1}/#{count})"
           message = ""
-          if e.response["errors"]
-            e.response["errors"].each do |field, errors|
+          if e.error_details["errors"]
+            e.error_details["errors"].each do |field, errors|
```

Comment:
You could golf this down a little and remove the conditional by normalising 

```ruby
Array(e.error_details["errors"]).each #...
```

But it's purely a matter of opinion.